docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,631 @@ license: |
+    </tr>
+    <tr>
+      <td><b>count</b>([<b>DISTINCT</b>] <i>expression1[, expression2</i>])</td>
+      <td>(any[, any])</td>
+      <td>If specified <code>DISTINCT</code>, returns the number of rows for which the supplied expression(s) are unique and not null; otherwise, returns the number of rows for which the supplied expression(s) are all not null.</td>
+    </tr>
+    <tr>
+      <td><b>count_if</b>(<i>predicate</i>)</td>
+      <td>expression that returns a boolean value</td>
+      <td>Returns the count number from the predicate evaluate to `TRUE` values.</td>
+    </tr> 
+    <tr>
+      <td><b>count_min_sketch</b>(<i>expression, eps, confidence, seed</i>)</td>
+      <td>(tinyint|int|bigint|smallint|string|binary, double, double, int)</td>
+      <td>`eps` and `confidence` are the double values between 0.0 and 1.0, `seed` is a positive integer. Returns a count-min sketch of a expression with the given `esp`, `confidence` and `seed`. The result is an array of bytes, which can be deserialized to a `CountMinSketch` before usage. Count-min sketch is a probabilistic data structure used for cardinality estimation using sub-linear space.</td>
+    </tr>
+    <tr>
+      <td><b>covar_pop</b>(<i>expression1, expression2</i>)</td>
+      <td>(double, double)</td>
+      <td>Returns the population covariance of a set of number pairs.</td>
+    </tr> 
+    <tr>
+      <td><b>covar_samp</b>(<i>expression1, expression2</i>)</td>
+      <td>(double, double)</td>
+      <td>Returns the sample covariance of a set of number pairs.</td>
+    </tr>  
+    <tr>
+      <td><b>{first | first_value}</b>(<i>expression[, `isIgnoreNull`]</i>)</td>
+      <td>(any[, boolean])</td>
+      <td>Returns the first value of expression for a group of rows. If isIgnoreNull is true, returns only non-null values, default is false. This function is non-deterministic.</td>
+    </tr>      
+    <tr>
+      <td><b>kurtosis</b>(<i>expression</i>)</td>
+      <td>double</td>
+      <td>Returns the kurtosis value calculated from values of a group.</td>
+    </tr>    
+    <tr>
+      <td><b>{last | last_value}</b>(<i>expression[, `isIgnoreNull`]</i>)</td>
+      <td>(any[, boolean])</td>
+      <td>Returns the last value of expression for a group of rows. If isIgnoreNull is true, returns only non-null values, default is false. This function is non-deterministic.</td>
+    </tr>      
+    <tr>
+      <td><b>max</b>(<i>expression</i>)</td>
+      <td>tinyint|short|int|bigint|float|double|date|timestamp|string, or arrays of these types</td>
+      <td>Returns the maximum value of the expression.</td>
+    </tr>          
+    <tr>
+      <td><b>max_by</b>(<i>expression1, expression2</i>)</td>
+      <td>tinyint|short|int|bigint|float|double|date|timestamp|string, or arrays of these types</td>
```

Comment:
Could you check again if all the input types are correct? `max_by`/`min_by` seems to accept null types and a struct of orderable element types?
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/MaxByAndMinBy.scala#L49
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ordering.scala#L101